mitosheet/src/extension.tsx

```diff
@@ -1,16 +1,27 @@
 // Copyright (c) Jupyter Development Team.
 
-// Entry point for the notebook bundle containing custom model definitions.
-//
-// Setup notebook base URL
+// This file contains the javascript that is run when the notebook is loaded.
```

Comment:
TODO: comment, cleanup, and organize these files so it is clear what is what. If we can move them into separate folders, this would be really nice!